src/adapters/kruxlink.js

```diff
@@ -43,7 +43,7 @@ function _makeCallback(id, placements) {
     // Add any no-bids remaining
     for (var placementCode in placements) {
```

Comment:
Please rename `placementCode` here to `impId` as that is what it is (and most likely which caused the error in the first place). Thanks!